src/applications/disability-benefits/all-claims/content/privateMedicalRecords.jsx

```diff
@@ -43,3 +43,95 @@ export const documentDescription = () => (
     </p>
   </div>
 );
+
+export const patientAcknowledgmentText = (
+  <AdditionalInfo triggerText="Read the full text.">
+    <h4>PATIENT AUTHORIZATION:</h4>
```

Comment:
I mentioned it in the other PR, but I'll mention it again here: This will need plan language edits before we release it into the wild, but it's okay as a placeholder for now.

Reply:
@Jennywalter Passing the note along to you as requested!

Reply:
Hi @cvalarida the presentment of the Patient Acknowledgement and the 'Read the full text' expansion was approved by Paul Shute and Andrea Schneider so it will need to stay as is for now.  This text was taken directly from the form so it cannot be modified.  Thanks again!